SUASSystem/SUASSystem/sda_converter.py

```diff
@@ -21,8 +21,7 @@ def __init__(self, initial_coordinates, fly_zones):
         self.initial_coordinates = initial_coordinates
         self.current_path = numpy.array([])
         self.current_path_index = 1
-        self.minimum_change_in_guided_waypoint = 3
-
+        self.minimum_change_in_guided_waypoint = 1000
```

Comment:
This is not correct. Please reduce this.

Reply:
lol done